### PR TITLE
Added Twitter lambda + Fixed https/http issue

### PR DESCRIPTION
**Pull request by Lourenc**

### CODE REVIEW

**Comment by Lourenc** [comment]
I've removed `.parameters.json` as there were sensetive data. Please, create your one to run it.

File: src/DeepAwsReinvent2015/Backend/resources.json [hunk 1]
```diff
@@ -6,5 +6,13 @@
 			"methods": ["POST"],
 			"source": "src/Email/Create"
 		}
+	},
```
Comment (AlexanderC) [inline src/DeepAwsReinvent2015/Backend/resources.json]: Please do NOT use tabs, use spaces instead!

File: src/DeepAwsReinvent2015/Frontend/js/app/angular/directives/youtube.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ angular.module(moduleName).directive('awsYoutube', ['$sce',function($sce) {
     link: (scope)=> {
       scope.$watch('code', (newVal)=> {
         if (newVal) {
-          scope.url = $sce.trustAsResourceUrl('http://www.youtube.com/embed/' + newVal);
+          scope.url = $sce.trustAsResourceUrl('https://www.youtube.com/embed/' + newVal);
```
Comment (AlexanderC) [inline src/DeepAwsReinvent2015/Frontend/js/app/angular/directives/youtube.js]: why not using //www.youtube.com/embed/?

File: src/DeepAwsReinvent2015/Frontend/js/app/angular/services/instagram.js [hunk 3]
```diff
@@ -34,7 +34,7 @@ class InstagramService {
    */
   getEmbedData(link) {
     let defered = this.$q.defer();
-    let endPoint = 'http://api.instagram.com/oembed?url=' + link + '&callback=JSON_CALLBACK';
+    let endPoint = 'https://api.instagram.com/oembed?url=' + link + '&callback=JSON_CALLBACK';
```
Comment (AlexanderC) [inline src/DeepAwsReinvent2015/Frontend/js/app/angular/services/instagram.js]: Why not using '//api.instagram.com/oembed?url=' + link + '&callback=JSON_CALLBACK'

File: src/DeepAwsReinvent2015/Backend/src/Twitter/Retrieve/Handler.es6 [hunk 4]
```diff
@@ -0,0 +1,49 @@
+/**
+ * Created by Korney Vasilchenko on 01/10/15
+ */
+
+'use strict';
+
+var Twit = require('twit');
+
+import DeepFramework from '@mitocgroup/deep-framework';
```
Comment (AlexanderC) [inline src/DeepAwsReinvent2015/Backend/src/Twitter/Retrieve/Handler.es6]: @mitocgroup/ is not used anymore, use 'deep-framework'

File: src/DeepAwsReinvent2015/Backend/src/Twitter/Retrieve/bootstrap.es6 [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+ * Created by Korney Vasilchenko on 01/10/15
+ */
+
+'use strict';
+
+if (typeof Symbol === 'undefined') {
+  require('babel-core/polyfill');
+}
+
+import DeepFramework from '@mitocgroup/deep-framework';
```
Comment (AlexanderC) [inline src/DeepAwsReinvent2015/Backend/src/Twitter/Retrieve/bootstrap.es6]: @mitocgroup/ is not used anymore, use 'deep-framework'

File: src/DeepAwsReinvent2015/Backend/src/Twitter/Retrieve/Handler.es6 [hunk 6]
```diff
@@ -0,0 +1,49 @@
+/**
+ * Created by Korney Vasilchenko on 01/10/15
+ */
+
+'use strict';
+
+var Twit = require('twit');
```
Comment (AlexanderC) [inline src/DeepAwsReinvent2015/Backend/src/Twitter/Retrieve/Handler.es6]: Why do you use such kind of requires?

use: import Twit from 'twit';

File: src/DeepAwsReinvent2015/Frontend/js/app/angular/directives/tweet.js [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+'format es6';
+
+import moduleName from '../name';
+
+angular.module(moduleName).directive('embedTweet', [()=> {
+  return {
+    restrict: 'E',
+    scope: { tweetId:'=' },
+    link: (scope, element, attrs)=> {
+      console.log(element, scope.tweetId);
```
Comment (AlexanderC) [inline src/DeepAwsReinvent2015/Frontend/js/app/angular/directives/tweet.js]: Please remove debug info

File: src/DeepAwsReinvent2015/Frontend/js/app/angular/services/twitter.js [hunk 8]
```diff
@@ -0,0 +1,55 @@
+'use strict';
+'format es6';
+
+import moduleName from '../name';
+
+let resource = DeepFramework.Kernel.get('resource');
```
Comment (AlexanderC) [inline src/DeepAwsReinvent2015/Frontend/js/app/angular/services/twitter.js]: do not use variables outside the class. The only exception in our case is the injection at the bottom

File: src/DeepAwsReinvent2015/Frontend/js/app/angular/services/twitter.js [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+
+  /** getTweetsByQuery
+   * @param {String} idt
+   * @returns {promise}
+   */
+  getOembedData(id) {
+    return this.twitterApiRequest('statuses/oembed', { id, });
+  };
+
+  twitterApiRequest(action, params) {
```
Comment (AlexanderC) [inline src/DeepAwsReinvent2015/Frontend/js/app/angular/services/twitter.js]: Add docblock

File: src/DeepAwsReinvent2015/Backend/src/Twitter/Retrieve/Exceptions/TwitterApiOperationException.es6 [hunk 10]
```diff
@@ -0,0 +1,19 @@
...
+'use strict';
+
+import {Exception} from './Exception';
+
+/**
+ * Thrown when operation on the database failed
+ */
+export class TwitterApiOperationException extends Exception {
+  /**
+   * @param error
```
Comment (AlexanderC) [inline src/DeepAwsReinvent2015/Backend/src/Twitter/Retrieve/Exceptions/TwitterApiOperationException.es6]: Add type hinting

File: src/DeepAwsReinvent2015/Backend/src/Twitter/Retrieve/Handler.es6 [hunk 11]
```diff
@@ -0,0 +1,49 @@
...
+export default class extends DeepFramework.Core.AWS.Lambda.Runtime {
+
+  /**
+   * @param {Array} args
+   */
+  constructor(...args) {
+    super(...args);
+  }
+
+  handle(request) {
```
Comment (AlexanderC) [inline src/DeepAwsReinvent2015/Backend/src/Twitter/Retrieve/Handler.es6]: Add docblock

File: src/DeepAwsReinvent2015/Backend/src/Twitter/Retrieve/Handler.es6 [hunk 12]
```diff
@@ -0,0 +1,49 @@
...
+        throw new DatabaseOperationException(err);
+      };
+
+      return this.createResponse(data).send();
+    });
+  };
+
+  /**
+   * Perform Twitter API call
+   * @param data
```
Comment (AlexanderC) [inline src/DeepAwsReinvent2015/Backend/src/Twitter/Retrieve/Handler.es6]: Add type hinting

File: src/DeepAwsReinvent2015/Frontend/js/app/angular/controllers/twitterCtrl.js [hunk 13]
```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+import moduleName from '../name';
+
+class twitterController {
+  constructor(Twitter, $scope) {
```
Comment (AlexanderC) [inline src/DeepAwsReinvent2015/Frontend/js/app/angular/controllers/twitterCtrl.js]: Add type hinting

**Pull request merged**